src/plugins/imagery/components/Compass/Compass.vue

```diff
@@ -26,19 +26,20 @@
     :style="compassDimensionsStyle"
 >
     <CompassHUD
-        v-if="shouldDisplayCompassHUD"
+        v-if="hasCameraFieldOfView"
         :heading="heading"
-        :roll="roll"
         :sun-heading="sunHeading"
-        :camera-field-of-view="cameraFieldOfView"
+        :camera-angle-of-view="cameraAngleOfView"
```

Comment:
You're right, it's not an accurate representation of field of view, and it's not configurable.